Resources/Prototypes/Entities/Objects/Tools/welders.yml

```diff
@@ -66,6 +66,11 @@
         - ReagentId: WeldingFuel
           Quantity: 1000
         maxVol: 1000
+  - type: Tool
+    useSound:
+      collection: Welder
+    qualities: Welding
+    speed: 1.5
```

Comment:
The experimental welder doesn't need a speed boost, since the main ability is that it regenerates its own fuel, but either way is fine.

Reply:
Did not know that, I'll remove the boost then. Thanks